### PR TITLE
Andrew/project page

### PR DESCRIPTION
**Pull request by andrewkalumba**
Project Page update

### CODE REVIEW

File: project-page.html [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+                    </li>
+                </ul>
+                <div class="legal">
+                    &copy; 2024 by Git-Project
+                </div>
+            </div>
+            <section class="main-content">
+                    <article>
+                        <div class="article__item">
+                            <h3 class="sub-header">Details the relevant git commands relevant to creating a project locally.</h3>
```
Comment (Svpriyaa2808) [inline project-page.html:73]: you can change the title "The relevant git commands to create a project locally"

File: project-page.html [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
+                            </table>
+                        </article>
+
+                        <article>
+                            <div class="article__item">
+                                <h3 class="sub-header">Details the git commands to link the local and remote repos</h3>
+                                <p>Step 1: Create a new directory,
+                                "mkdir my-first-git-project". This command creates a new directory named my-first-git-project.</p>
+                                <p>Step 2: Navigate to the project directory "cd my-git-project". Changes into the newly created directory.</p>
+                                <p>Step 3: Initialize a Git repository, "git init". This command initializes a new Git repository in the current directory.</p>
```
Comment (Svpriyaa2808) [inline project-page.html:274]: you can check for alignment

File: project-page.html (hunk 1 above)
Comment (andrewkalumba) [reply project-page.html:73]: okay😜

File: project-page.html [hunk 3]
```diff
@@ -0,0 +1,395 @@
...
+                                
+                                To create a blank project, under "Start from scratch", click Table, Roadmap, or Board.
+                                To create a project from a template, click the built-in template that you want to use.
+                                Optionally, if you selected a template, review the fields, views, workflows, and insights that will be created.
+                                
+                                In the text box under "Project name", type a name for your new project.
+                                
+                                Click Create project.</p>
+                                
+                                <p>Setting your project description and README
```
Comment (Svpriyaa2808) [inline project-page.html:360]: you can check for alignment in all paragraph tag

File: project-page.html [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+                                    <td></td>
+                                    <td>git tag <tag-name></td>
+                                    <td>Creates a tag for a specific commit (useful for releases).</td>
+                                </tr>
+                            </table>
+                        </article>
+
+                        <article>
+                            <div class="article__item">
+                                <h3 class="sub-header">Details the git commands to link the local and remote repos</h3>
```
Comment (Svpriyaa2808) [inline project-page.html:270]: The h3 can be changed "The git commands to link the local and remote repo"

File: project-page.html [hunk 5]
```diff
@@ -0,0 +1,395 @@
...
+                                   git push -u origin main
+                                   Now, your local changes are sent to the remote repository, and the two repositories are linked. You can subsequently use git pull to fetch changes from the remote repository or continue pushing changes with git push.
+                                </p>
+                            </div>
+                        </article>
+
+                        <article>
+                            <div class="article__item">
+                                <h3 class="sub-header">why remote repositories are necessary</h3>
+                                <p>Remote repositories play a crucial role in the world of version control. In simple terms, a remote repository refers to the shared storage where developers can collaborate, store, and access their codebase. It serves as a central hub for a project, allowing team members to work together seamlessly.</p>
```
Comment (VaishaliGovindaraj) [inline project-page.html:299]: Check on the alignment :)

**Review by TobiasKjernell — APPROVED** [review]

**Pull request merged**